### PR TITLE
Updated the product list with new categories

### PR DESCRIPTION
**Pull request by hardikhdholariya**

### CODE REVIEW

File: cyences_app_for_splunk/bin/cs_product_list.py [hunk 1]
```diff
@@ -468,6 +472,23 @@ def build_app_dependency_search(app_list):
             }
         ],
     },
+    {
+        "name": "Vulnerability",
+        "app_dependencies": [],
+        "metadata_count_search": '`cs_vulnerabilities_indexes` sourcetype="*vuln*" | stats count',
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/bin/cs_product_list.py:478]: Sourcetype may not contain *vuln*
We need to stick with what we are using in the datamodel, alert or dashboard.

File: cyences_app_for_splunk/bin/cs_product_list.py [hunk 2]
```diff
@@ -323,6 +287,46 @@ def build_app_dependency_search(app_list):
...
+                "earliest_time": "-7d@d",
+                "latest_time": "now",
+            }
+        ],
+    },
+    {
+        "name": "Network",
+        "app_dependencies": [],
+        "metadata_count_search": '`cs_network_indexes` tag=network tag=communicate | stats count',
+        "macro_configurations": [
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/bin/cs_product_list.py:317]: This stuff would be very useful

File: cyences_app_for_splunk/default/macros.conf [hunk 3]
```diff
@@ -1007,6 +1007,11 @@ definition = NOT activityDisplayName IN ("ApprovalNotification_Create", "Feature
 iseval = 0
 
 # Email
+[cs_email_indexes]
+definition = `cs_o365` OR `cs_gws`
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/macros.conf:1011]: Let's use this macro everywhere related to email.

File: cyences_app_for_splunk/default/macros.conf [hunk 4]
```diff
@@ -1040,6 +1045,11 @@ iseval = 0
 definition = search *
 iseval = 0
 
+[cs_network_indexes]
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/macros.conf:1048]: Let's use this macro everywhere we are using network related stuff.

**Review by VatsalJagani — CHANGES_REQUESTED** [review]

File: cyences_app_for_splunk/default/macros.conf (hunk 4 above)
Comment (hardikhdholariya) [reply cyences_app_for_splunk/default/macros.conf:1048]: For network, we are using "Network Traffic" datamodel and index used from CIM app `cim_Network_Traffic_indexes`

Can we use it directly?

Comment (VatsalJagani) [reply cyences_app_for_splunk/default/macros.conf:1048]: Yes, but we are not using keywork **cim** anymore right??

Comment (hardikhdholariya) [reply cyences_app_for_splunk/default/macros.conf:1048]: yes but the network traffic model we are using from CIM app so all alerts internally using the `cim_Network_Traffic_indexes` macro.

Comment (VatsalJagani) [reply cyences_app_for_splunk/default/macros.conf:1048]: ohh okay, that macro is not present in Cyences then? If not present then it is not recommend to change from our app

**Review by VatsalJagani — APPROVED** [review]

**Pull request merged**